bundles/org.openhab.binding.miio/src/main/resources/ESH-INF/thing/vacuumThing.xml

```diff
@@ -127,6 +129,14 @@
 		<category>Energy</category>
 		<state pattern="%.0f" readOnly="true" />
 	</channel-type>
+	<channel-type id="water_box_status">
+		<item-type>Switch</item-type>
+		<label>Water Box State</label>
+	</channel-type>
+	<channel-type id="water_box_carriage_status">
+		<item-type>Switch</item-type>
+		<label>Water Box Carriage State</label>
+	</channel-type>
```

Comment:
Can these channels set values? If not, maybe it's better to mark them as read only.